app/assets/stylesheets/sulCollection.scss

```diff
@@ -97,6 +97,10 @@
   font-size: 28px;
 }
 
+label.toggle-bookmark {
```

Comment:
Should this be set in sulBase.scss instead since it's used beyond just the collection page?

Reply:
Sounds good!